_includes/tess.html

```diff
@@ -0,0 +1,3 @@
+<a class="no_icon nohover" data-toggle="tooltip" title="TeSS Search: {{include.search}}" href="https://tess.elixir-europe.org/search?q={{include.search}}">
+  <img style="background-color: orange;" class="registry_logo" src="images/tess-logo.png" alt="TeSS logo"/>
```

Comment:
```suggestion
  <img style="background-color: #f47d20;" class="registry_logo" src="images/tess-logo.png" alt="TeSS logo"/>
```

Reply:
I would differentiate here from the color from bio.tools. I took the same color from the website  of tess